src/tests/Microsoft.PowerFx.Interpreter.Tests/DatabaseSimulationTests.cs

```diff
@@ -22,11 +23,13 @@ namespace Microsoft.PowerFx.Interpreter.Tests
     public class DatabaseSimulationTests
     {
         [Theory]
-        [InlineData("Patch(Table, First(Filter(Table, MyStr=\"Str3\")), {MyDate: \"2022-11-14 7:22:06 pm\"})", false)]
-        [InlineData("Patch(Table, First(Filter(Table, MyStr=\"Str3\")), {MyDate: DateTimeValue(\"2022-11-14 7:22:06 pm\") })", true)]
-        public void DatabaseSimulation_Test(string expr, bool checkSuccess)
+        [InlineData("Patch(Table, First(Filter(Table, MyStr = \"Str3\")), {MyDate: \"2022-11-14 7:22:06 pm\"})", false, 0)]
+        [InlineData("Patch(Table, First(Filter(Table, MyStr = \"Str3\")), {MyDate: DateTimeValue(\"2022-11-14 7:22:06 pm\") })", true, 0)]
```

Comment:
>DateTimeValue(\"2022-11-14 7:22:06 pm\") [](http://example.com/codeflow?start=83&length=40)

Not your changes, but can you update it to `DateTime(2022,11,14,19,22,6)`, to make this test more non-en-US friendly?